src/AWS/CRT/HTTP/Headers.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace AWS\CRT\HTTP;
+
+use AWS\CRT\Internal\Encoding;
+
+final class Headers {
+    private $headers = [];
```

Comment:
No need to set this here cause it's set in the constructor.  Won't hurt anything though, just a bit redundant